classes/class-wc-rest-connect-shipping-rates-controller.php

```diff
@@ -71,7 +71,7 @@ public function update_items( $request ) {
 		$payload[ 'carrier' ] = 'usps';
 
 		// Exclude extraneous package fields
-		$whitelist = array_fill_keys( array( 'length', 'width', 'height', 'weight', 'template' ), true );
+		$whitelist = array_fill_keys( array( 'length', 'width', 'height', 'weight', 'template', 'service_id' ), true );
```

Comment:
Will this handle the case when a package has been changed, or blow up the server's request validation?

I also moved this logic to JS in my PR: https://github.com/Automattic/woocommerce-connect-client/pull/683/files#diff-d8b04016a3889c977c51e0ed0ae5a2f7 . It was basically the same when requesting rates and when purchasing labels. Just FYI to avoid merge conflicts.